src/api/system.c

```diff
@@ -600,7 +698,15 @@ static int f_mkdir(lua_State *L) {
   const char *path = luaL_checkstring(L, 1);
 
 #ifdef _WIN32
-  int err = _mkdir(path);
+  LPWSTR wpath = utfconv_utf8towc(path);
+  if (path == NULL) {
```

Comment:
Did you meant `wpath` instead of `path`?

Reply:
Thanks for catching that!